pyproject.toml

```diff
@@ -50,7 +50,7 @@ jobflow = ["jobflow[fireworks]>=0.1.14", "jobflow-remote>=0.1.0"]
 mlp = ["matgl>=1.1.2", "chgnet>=0.3.3", "mace-torch>=0.3.3", "torch-dftd>=0.4.0"]
 mp = ["atomate2>=0.0.14"]
 newtonnet = ["newtonnet>=1.1"]
-parsl = ["parsl[monitoring]>=2024.5.27; platform_system!='Windows'"]
```

Comment:
Please don't change unrelated versions. They should work just fine.

Reply:
I don't know what was changed here. I see that this file is the same as the one in Quantum-Accelerator: main. Maybe you changed it back already.

Reply:
Did I change the parsl version from 2023.10.23 to 2.24.5.27?

Reply:
Yes, I took care of it. However, there are several other changes that still need to be reverted back, as shown in the file comparison view.